### DISCUSSION TITLE
uYouPlus/uYouEnhanced Issues

### DISCUSSION THREAD

**Discussion by arichornlover** [arichornlover/uYouEnhanced#6] [General]
a topic where issues are severe and found in this discussion.

**Comment by arichornlover** [comment]
(uYouEnhanced Issue) Low Contrast Mode isn’t working as of right now!
YouTube changed the way with how UIColor works, so if the tweak doesn’t work when it’s on, Blame YouTube.
![A9B36021-0C4E-4B06-BA1A-896E194B9600](https://user-images.githubusercontent.com/78001398/200179995-9b0bcd76-a5ad-47df-949e-d97d3c34d408.png)

**Comment by arichornlover** [comment]
New issue, as I would of expected.
In 17.46.4-2.1_F2 (Pre-release 8)
YouTube will randomly crash if you go everywhere around the app browsing, watching videos & downloading uYou content. This is why it was in a pre-release state in the first place. this type of crash issue is caused by PoomSmart’s YouTube Return Dislike Tweak. Read the crash log below to understand what I’m talking about.
[YouTube-2022-12-03-020329.txt](https://github.com/arichornlover/uYouEnhanced/files/10145969/YouTube-2022-12-03-020329.txt)

**Comment by arichornlover** [comment]
Small little problem. I was gonna release v18 of YouTube but for some dang reason I built the app and tried using it and all I’ve get is app corruption (app corruption does this: pressing the app fades and takes you back into the Home Screen)
How is this happening? For me personally I have no clue I either think it’s caused by LowContrastMode, uYou 2.3~1 or YTUHD but don’t clearly know yet. If I can’t fix this issue then I will have to take a break from uYouEnhanced. (These Issues occurs in 18.01.2-18.01.4 or newer)

**Reply by arichornlover** [reply]
Seems like the corruption issue is fixed in v18.01.6.
I’m that dumb huh.

**Comment by thaibm1992** [comment]
You are amazing in developing uYouEnhanced, I am always watching and learning from you. However, the work below is not live yet, are you continuing to develop it?
- Hide uYou Tab doesn't work
- Disable automatic updates uYou doesn't work

**Reply by arichornlover** [reply]
Hide uYou Tab is in testing
Disable automatic updates isn't my problem. ask qn about that.
